go-controller/pkg/ovn/egressip_test.go

```diff
@@ -1653,26 +1397,11 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 						},
 					},
 				}
-				fakeOvn.startWithDBSetup(ctx, libovsdbtest.TestSetup{
-					NBData: []libovsdbtest.TestData{
-						&nbdb.LogicalRouter{
-							Name: ovntypes.OVNClusterRouter,
-							UUID: ovntypes.OVNClusterRouter + "-UUID",
-						},
-						&nbdb.LogicalRouter{
-							Name: ovntypes.GWRouterPrefix + node1.Name,
-							UUID: ovntypes.GWRouterPrefix + node1.Name + "-UUID",
-						},
-						&nbdb.LogicalRouter{
-							Name: ovntypes.GWRouterPrefix + node2.Name,
-							UUID: ovntypes.GWRouterPrefix + node2.Name + "-UUID",
-						},
-					},
-				})
-				fakeOvn.fakeExec.AddFakeCmdsNoOutputNoError(
-					[]string{
-						fmt.Sprintf("ovn-nbctl --timeout=15 set logical_switch_port etor-GR_node1 options:nat-addresses=router"),
-						fmt.Sprintf("ovn-nbctl --timeout=15 set logical_switch_port etor-GR_node2 options:nat-addresses=router"),
```

Comment:
Here you are removing these checks and never replacing them.

Reply:
I.e: please validate that these switch ports have that option configured at the end of the test

Reply:
done